server/src/main/resources/org/opensearch/bootstrap/security.policy

```diff
@@ -93,6 +93,26 @@ grant codeBase "${codebase.reactor-core}" {
   permission java.net.SocketPermission "*", "connect,resolve";
 };
 
+// security
+grant {
+    permission java.lang.RuntimePermission "accessClassInPackage.sun.security.internal.spec";
```

Comment:
nit: this block and the block below can be consolidated into a single `grant` block and the comment could be kept for maintainability to group permissions.

I don't expect this comment to be addressed, because JSM is on the deprecation path and there is an RFC open on whether it should be disabled in OpenSearch 3.0.0 entirely: https://github.com/opensearch-project/OpenSearch/issues/17181